docker/vttestserver/run_local.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
@rsajwani you were going to remove this file from here.
If we want to show it as an example we should move it to the examples folder.

Reply:
ah it was removed but got resurrected again.. let me remove it.